### ISSUE TITLE
validation score of VQA

### ISSUE THREAD

**Issue by Yucheng-Han** [airsplay/lxmert#106]
I downloaded the pre-trained model, and fine-tuned the VQA tasks. But I found while fine-tuning, the validation scores of VQA decreases in the first 4 epochs. Is this normal? I think it is hard to believe that the validation scores decrease but test scores increase...

**Comment by Yucheng-Han** [comment]
I am using PyTorch 1.7, and I would check whether PyTorch version is the cause. But if so, I think maybe the mask has something to do with the results.

**Comment by Yucheng-Han** [comment]
seems to be my own reason
